src/frontend/tests/e2e/specs/testWallet.js

```diff
@@ -0,0 +1,19 @@
+// https://docs.cypress.io/api/introduction/api.html
+
+describe('Loads the web app', () => {
+  context('1080p resolution', () => {
+    beforeEach(() => {
+      // run these tests as if in a desktop
+      // browser with a 1080p monitor
+      cy.viewport(1920, 1080)
+    })
+
+    it('Visits the app root url', () => {
```

Comment:
I think this is the same test as testLogin.js! Make sure to test if wallet header is visible!

Reply:
Oops, forgot to update the test. Fixed!